install/install.sh

```diff
@@ -13,7 +13,7 @@
 
 export PATH=$PATH:$ANSIBLE_HOME/bin
 PLAYBOOKFILE="qpc_playbook.yml"
-RELEASE_TAG='-e RELEASE_TAG=1.0.0'
+RELEASE_TAG='-e RELEASE_TAG=master'
```

Comment:
This change is to mach the version the master pipeline is creating, see https://github.com/quipucords/quipucords/blob/master/Jenkinsfile#L1

Better to name it master than 1.0.0 to not confuse with releases instead of the dev version